pyproject.toml

```diff
@@ -106,11 +103,8 @@ dev_all = [
     "autodoc-pydantic",
     "black",
     "coverage",
-    "flake8",
-    "isort",
```

Comment:
Since we are removing isort, the makefile should get updated as well.  It's not required for this PR, but if you can that'd be good.

Reply:
Good catch, updated!